packages/webapp/src/components/CreateUserAccount/index.jsx

```diff
@@ -130,21 +130,21 @@ export default function PureCreateUserAccount({ onSignUp, email, onGoBack }) {
         data-cy="createUser-language"
         control={control}
         name={LANGUAGE}
-        render={({ field: { onChange, onBlur, value } }) => (
-          value && setLanguage(value.value),
-          (
-            <ReactSelect
-              label={t('CREATE_USER.LANGUAGE_PREFERENCE')}
-              options={languageOptions}
-              onChange={onChange}
-              value={languageOptions[languageOption]}
-              style={{ marginBottom: '28px' }}
-              defaultValue={{
-                value: t('CREATE_USER.DEFAULT_LANGUAGE_VALUE'),
-                label: t('CREATE_USER.DEFAULT_LANGUAGE'),
-              }}
-            />
-          )
+        render={({ field: { onChange } }) => (
+          <ReactSelect
```

Comment:
Setting the state (calling `setLanguage`) was moved to the `onChange` handler to address this console warning:

![Screenshot 2023-11-07 at 1 44 11 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/6c28e973-0385-4630-8740-672358f9e614)